src/Program.ts

```diff
@@ -283,36 +283,38 @@ export class Program {
      * Get the type definitions for the specified file.
      * The results are cached for future performance boosts.
      */
-    private async getTypeDefinitionsForFile(pathAbsolute: string) {
+    private async getTypedefsForFile(pathAbsolute: string) {
         //you can only get type definitions for .brs files
-        if (!pathAbsolute.endsWith('.brs')) {
+        if (!pathAbsolute.toLowerCase().endsWith('.brs')) {
             return;
         }
 
-        let typeDefinitionPath = pathAbsolute.replace(/.brs$/, '.d.bs');
-
+        let typedefPath = pathAbsolute.replace(/.brs$/i, '.d.bs');
+        let lowerTypedefPath = typedefPath.toLowerCase();
```

Comment:
Maybe `typedefKey` instead? You could use another algorithm here.

Reply:
Good suggestion. I'll do that.